README.md

```diff
@@ -26,6 +26,12 @@ Create an HPI file to install in Jenkins (HPI file will be in `target/slack.hpi`
 
     mvn package
 
+If you are having trouble getting mvn to package due to a test failure, please add this to your pom.xml
+
+	<properties>
+		<skipTests>true</skipTests>
+	</properties>
```

Comment:
This should be surrounded by back ticks.  e.g.

```xml
<properties>
    <skipTests>true</skipTests>
</properties>
```

Also, using tabs instead of 4 spaces.  :-1: